model/trace/database.yaml

```
@@ -25,6 +25,13 @@ groups:
       - ref: db.instance.id
         requirement_level:
           recommended: If different from the `server.address`
+      - ref: db.collection.name
+        requirement_level:
+          conditionally_required: >
+            If readily available. Otherwise, when the instrumentation library supports parsing `db.statement` and it's enabled,
+            `db.collection.name` SHOULD be set to the first collection name found in the query.
```

(my thought is to get rid of "and it's enabled" in order to avoid giving the impression that parsing should be opt-in)

```suggestion
            If readily available. Otherwise, if the instrumentation library parses `db.statement` to capture
            `db.collection.name`, then it SHOULD be the first collection name found in the query.
```